lib/emscripten/src/lib.rs

```diff
@@ -545,6 +545,8 @@ pub fn generate_emscripten_env(globals: &mut EmscriptenGlobals) -> ImportObject
         "_sysconf" => func!(crate::env::_sysconf),
         "_getaddrinfo" => func!(crate::env::_getaddrinfo),
         "_times" => func!(crate::env::_times),
+        "_pathconf" => func!(crate::env::_fpathconf),
```

Comment:
`pathconf()` doesn't have the same signatura or functionality than `fpathconf()`, first accept a path and second one a file descriptor.